src/main/java/com/minecolonies/commands/CitizensCommand.java

```diff
@@ -0,0 +1,36 @@
+package com.minecolonies.commands;
+
+import com.google.common.collect.ImmutableMap;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Map;
+
+/**
+ * Minecolonies root command.
+ * <p>
+ * Manages all sub commands.
+ */
+public class CitizensCommand extends AbstractSplitCommand
+{
+    private static final String DESC = "citizens";
+
+    private final ImmutableMap<String, ISubCommand> subCommands =
+      new ImmutableMap.Builder<String, ISubCommand>()
+        .put("list", new ListCitizens("mc", DESC, "list"))
```

Comment:
Get "mc" or "minecolonies" from the public field in the parent command. And maybe list from the ListCommand